lib/src/r_tree/leaf_node.dart

```diff
@@ -58,6 +58,6 @@ class LeafNode<E> extends Node<E> {
 
```

Comment:
🧵 There are two instances in both `non_leaf_node_test.dart` and `leaf_node_test.dart` I'd update as well.